src/components/Tabset/index.js

```diff
@@ -88,30 +73,27 @@ export default class Tabset extends Component {
 
     updateTabsetButtonsVisibility() {
         const tabset = this.tabsetRef.current;
-        const tabsetScrollWidth = tabset.scrollWidth;
-        const tabsetScrollLeft = tabset.scrollLeft;
+        const {
+            scrollWidth: tabsetScrollWidth,
+            scrollLeft: tabsetScrollLeft,
+            offsetLeft: tabsetPosition,
+            offsetWidth: tabsetWidth,
+            children: tabsetChildren,
+        } = tabset;
```

Comment:
I think we should leave the names as default, there is not need to prefix them with tabset